src/main/java/frc/robot/subsystems/swerve/ModuleUtils.java

```diff
@@ -20,6 +21,15 @@ public static double toDriveDistance(double revolutions) {
         return Conversions.revolutionsToDistance(revolutions, ModuleConstants.WHEEL_DIAMETER_METERS);
     }
 
+    public static Translation2d getModulePositionRelativeToMiddleOfRobot(ModuleName moduleName){
+        return switch (moduleName) {
+            case FRONT_LEFT ->  new Translation2d(SwerveConstants.FRONT_LEFT_TRANSLATION2D.getX(), SwerveConstants.FRONT_LEFT_TRANSLATION2D.getY());
```

Comment:
```suggestion
            case FRONT_LEFT ->  new Translation2d(SwerveConstants.FRONT_LEFT_TRANSLATION2D);
```

im pretty sure you can do this

Reply:
you can't